src/object.d

```diff
@@ -1731,6 +1731,19 @@ class Throwable : Object
             }
         }
     }
+
+    /**
+     * Get the message describing the error.
+     * Base behavior is to return the `Throwable.msg` field.
+     * Override to return some other error message.
+     *
+     * Returns:
+     *  Error message
+     */
+    @__future const(char)[] message() const
+    {
+        return this.msg;
```

Comment:
Need a test case to turn the red to green.

Reply:
I see all tests in green, or do you mean the green of a review approval?

Also, there are some test cases already included, is there anything in particular you see missing?

Reply:
Line 1745 above is red. It isn't executed because all the test cases override `Throwable.message()`. Need a test case that does not override it, and just calls it.

Reply:
@leandro-lucarella-sociomantic There is a CodeCov browser extension, which overlays the unittest coverage on top of the diff for pull requests. I think what Walter is saying is that this line doesn't show up as covered by a unittest.

Reply:
@ZombineDev  is right. Here it is: https://github.com/codecov/browser-extension Please install it!

Reply:
Thanks, had no idea there's an extension, will do it today!

Reply:
I've started looking into this and it feels something is off with the code coverage - because this is already covered by: https://github.com/dlang/druntime/pull/1895/files#diff-8289be999021da628af8f313e9992573R4 (I even confirmed this with printouts in the line - it sure gets triggered)

Could it be that source of the problem is that code coverage doesn't collect ones from the test suite? IIRC, we have the similar problem in sociomantic's ocean, about the code coverage overriding each other, I wonder if it same here.

Reply:
I've added the unittest, let's see if that helps

Reply:
Ok, green now!

Reply:
> [..] it feels something is off with the code coverage - because this is already covered [..]

Not sure if there's a bugzilla issue about this (if not there should be), the reason for this initially not appearing covered is that coverage in external test files is essentially ignored. I think it is a build-system level issue, though I haven't looked in detail.